<?xml version="1.0"?>
<launch>
  <arg name="transport" default="udp"/>
  <arg name="scanner_ip" default="10.0.10.76"/>
  <arg name="device" default="R2300"/>

  <!-- R2000 Driver -->
  <node pkg="pf_driver" type="ros_main" name="r2300_node" args="$(arg transport) $(arg scanner_ip) 0 $(arg device)" output="screen"/>

  <node pkg="tf" type="static_transform_publisher" name="ring1" args="0 0 0 0 0.0785 0 /base_link /scanner_1 10" />
	<node pkg="tf" type="static_transform_publisher" name="ring2" args="0 0 0 0 0.0262 0 /base_link /scanner_2 10" />
	<node pkg="tf" type="static_transform_publisher" name="ring3" args="0 0 0 0 -0.0785 0 /base_link /scanner_3 10" />
	<node pkg="tf" type="static_transform_publisher" name="ring4" args="0 0 0 0 -0.0262 0 /base_link /scanner_4 10" />
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find pf_driver)/rviz/r2300.rviz" />
</launch>